### PR TITLE
Mrazadar/PON-14/checkout-page-design-edits

### PR DESCRIPTION
**Pull request by mrazadar**
- Checkout page design edits and copy changes
- refactor: removed the new 18n translations files
- test: fixed the failing tests
- feat: removed the fraction zero digits 
- feat: better error handling 
- fix: Button states fixed

### CODE REVIEW

**Review by NawfalAhmed — APPROVED** [review]

File: src/subscription/alerts/SubscriptionAlerts.jsx [hunk 1]
```diff
@@ -15,8 +15,8 @@ export const SubscriptionAlerts = () => (
   <AlertList
     messageCodes={{
       empty_subscription: (<EmptySubscriptionMessage />),
-      'embargo-error': (<EmptySubscriptionMessage />),
-      'basket-changed-error': (<BasketChangedError />),
+      embargo_error: (<EmptySubscriptionMessage />),
```
Comment (julianajlk) [inline src/subscription/alerts/SubscriptionAlerts.jsx:18]: Can we add `<FallbackErrorMessage />` instead until Product decides what we should display to the learner in this case?

**Review by julianajlk — APPROVED** [review]
LGTM! Curious on the reason to change `confirmation_status` to `status`?

File: src/subscription/alerts/SubscriptionAlerts.jsx (hunk 1 above)
Comment (mrazadar) [reply src/subscription/alerts/SubscriptionAlerts.jsx:18]: > LGTM! Curious on the reason to change `confirmation_status` to `status`?

https://2u-internal.atlassian.net/browse/PON-226

Comment (mrazadar) [reply src/subscription/alerts/SubscriptionAlerts.jsx]: > Can we add `<FallbackErrorMessage />` instead until Product decides what we should display to the learner in this case?

done

**Pull request merged**